docs/webgl/reference.html

```diff
@@ -1140,6 +1140,14 @@ <h2>enum <a id='CompressedPixelDataType' href='#CompressedPixelDataType'>Compres
 <li>DXT1_RGBA</li>
 <li>DXT3_RGBA</li>
 <li>DXT5_RGBA</li>
+<li>RED_RGTC1</li>
```

Comment:
I wasn't sure if this file has to be kept up-to-date. Let me know if I should revert